activesupport/lib/active_support/core_ext/name_error.rb

```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 class NameError
+  attr_writer :receiver
```

Comment:
I think we may want to `:nodoc:` this, unless you see some other use that people could have with it?